### DISCUSSION TITLE
Post featured images and thumbnails on homepage

### DISCUSSION THREAD

**Discussion by Kirjakovski** [adityatelange/hugo-PaperMod#711] [Q&A]
Dear all,

I've spend several hours googling on how to add featured image to posts and then this image to be shown in the posts list on the homepage as thumbnails. I'm using PaperMod theme. Anyone willing to help me out?

**Comment by yvoictra** [comment]
Hello, 

Did you check this?

https://adityatelange.github.io/hugo-PaperMod/posts/papermod/papermod-features/#post-cover-image

Best Regards

**Reply by Kirjakovski** [reply]
Thank you for taking the time to answer. Yes, I've seen it but since I'm quite new to the Hugo site development, it is rather a little confusing to me. Do I add that code to the front matter of the .md post for example? Or I need to intervene in the config file?

**Reply by Kirjakovski** [reply]
I think I understood how to show the cover images.

**Reply by yvoictra** [reply]
> Thank you for taking the time to answer. Yes, I've seen it but since I'm quite new to the Hugo site development, it is rather a little confusing to me. Do I add that code to the front matter of the .md post for example? Or I need to intervene in the config file?

Just to use it in the .md post file.

Best Regards.
